src/erf.jl

```diff
@@ -20,3 +19,41 @@ function erfc(a::Interval{T}) where T
     isempty(a) && return a
     @round( erfc(a.hi), erfc(a.lo) )
 end
+
+function erfinv(a::BigFloat)
```

Comment:
You should define versions with `erfinv(a::BigFloat, r::RoundingMode)`.
Then you can use `@round` as you suggested for the interval version.